ios/CustomerioReactnative.swift

```diff
@@ -197,17 +194,11 @@ class CustomerioReactnative: NSObject {
         current.getNotificationSettings(completionHandler: { permission in
             switch permission.authorizationStatus  {
             case .authorized:
-                status = .authorized
+                status = .granted
             case .denied:
                 status = .denied
             case .notDetermined:
                 status = .notDetermined
-            case .ephemeral: // authorized to send or receive notifications for limited time
-                // @available(iOS 14.0, *)
-                status = .ephemeral
-            case .provisional: //authoized to push non-interuptive notifications
-                // @available(iOS 12.0, *)
-                status = .provisional
             default:
                 status = .unknown
             }
```

Comment:
The value of the function `CustomerioTracking.getPushPermissionStatus` is to determine if we should show the push prompt or not. Determine if we send a push to this device, will a push notification show up on the device. 

Therefore, I disagree that `ephemeral` is unknown, it should be `.granted` since `ephemeral` means push permission is authorized, but for a limited time. 

`.provisional` should be `granted` because push permission has been authorized, but the behavior of the push is unique. [Learn more](https://phiture.com/blog/provisional-push-what-is-it-and-how-will-it-impact-your-addressable-audience/) about provisional push behavior on a device.

Reply:
I do agree that there should be a `default` case to stay backwards compatible with future iOS versions that might add more cases to `UNAuthorizationOptions` without our SDK being updated. 

I think that `notDetermined` and `unknown` are 2 words that are very similar to one another. I am leaning towards `default` case returning `notDetermined` and removing `unknown` case from the enum class.

Reply:
Made the changes in latest commit. thanks!